modules/tinymce/src/plugins/table/test/ts/browser/command/InsertTableCommandTest.ts

```diff
@@ -13,69 +13,82 @@ UnitTest.asynctest('browser.tinymce.plugins.table.command.InsertTableCommandTest
   TinyLoader.setup((editor, onSuccess, onFailure) => {
     const tinyApis = TinyApis(editor);
 
+    const sResetContent = Step.sync(() => editor.resetContent(''));
+    const sAssertNumNewUndoLevels = (expected: number) => Step.sync(() => {
```

Comment:
I will note I found this naming a bit confusing because it sort of implies some kind of state (like if I call it once, then change something, and call it again, I'd get the difference rather than the length of all undo levels except the first)

Reply:
Open to suggestions for a better name? The idea here was to check the number of new undo levels from the initial state (which will always have 1 level).

Reply:
Some ideas:
1. We could just make it `sAssertNumUndoLevels` and add 1 to each number we assert it with
2. Could we tie a TestStore to undo events (if we have those) that way we can call `.sClear` and `.sAssert` as needed

Reply:
> We could just make it sAssertNumUndoLevels and add 1 to each number we assert it with
I originally did this, but it was just confusing to me.

> Could we tie a TestStore to undo events (if we have those) that way we can call .sClear and .sAssert as needed
We don't have this. @JamesToohey created a JIRA for us to look into adding this the other day.

Reply:
Only thing I can think of is `sAssertNumTestUndoLevels` and I'm not sure that's any better.